src/System.Reflection.Metadata/ref/System.Reflection.Metadata.cs

```diff
@@ -3135,6 +3138,7 @@ public enum Machine : ushort
         AM33 = (ushort)467,
         Amd64 = (ushort)34404,
         Arm = (ushort)448,
+        Arm64 = (ushort)43620,
```

Comment:
@davidwrighton, this was added here - https://github.com/dotnet/corefx/pull/26243

This is expected, correct?